### DISCUSSION TITLE
How to account for spread in backtesting.py for EURUSD trading strategy?

### DISCUSSION THREAD

**Discussion by masterkey2000** [kernc/backtesting.py#921] [Q&A] [answered]
I am interested in backtesting a trading strategy for EURUSD using backtesting.py. I have the Bid price data, and I would like to know how to account for the spread (the constant difference between the Ask and Bid price) in the backtesting process. Could you please provide guidance on how to do this?

**Comment by gowestyang** [comment, marked as answer]
My idea is to set a higher commission
